client/app/pages/users/show.js

```diff
@@ -4,9 +4,10 @@ import { absoluteUrl } from '@/services/utils';
 import template from './show.html';
 import './settings.less';
 
+const regenerateConfirm = { class: 'btn-warning', title: 'Regenerate' };
```

Comment:
Why define it so far from where it's used?

Reply:
I referred to `deleteConfirm` in  https://github.com/getredash/redash/blob/v6.0.0/client/app/pages/data-sources/show.js#L6 as a reference. But `regenerateConfirm` is used only once in `regenerateUserApiKey`. Should I write as an inline?